### ISSUE TITLE
Hidden ads in docker release v3.1.0?!?!

### ISSUE THREAD

**Issue by GameWatchProjects** [TasmoAdmin/TasmoAdmin#959]
Hello,
can anyone confirm or re reproduce the case, when using the docker release v3.1.0 that a piHole (Pi-hole [v5.17.1](https://github.com/pi-hole/pi-hole/releases/v5.17.1) FTL [v5.23](https://github.com/pi-hole/FTL/releases/v5.23)) blocks permanent the urls "ad.doubleclick.net" and "www.googleadservices.com", and is calling the urls every minute?

I just happened to see it when looking at the pihole query log.

When i downgrade the Docker release to 3.0.8, these urls will not shown in the pihole query log, but when i switch to v3.1.0  the urls shown again in the pihole query log. It can not been an other docker container do this, while the tasmoadmin have in my case an explicit owen local url and this url is shown as referrer in the query logs.

**Comment by inverse** [comment]
Seems very odd that this would be happening - are you able to isolate it to the container itself?

Here's the  changelogs between those versions. https://github.com/TasmoAdmin/TasmoAdmin/compare/v3.0.8...v3.1.0

**Comment by mafrei2019** [comment]
Seiten Sind bei mir auch blockeriert TasmoAdmin v3.1.1 und PiHole Docker Tag [2023.05.2] Pi-hole [v5.17.1] FTL v5.23) Web Interface [v5.20.1].
Ich dachte aber das ist von PiHole und nicht durch TA. Wenn ich den DNS nicht über PiHole lasse funktionieren die Seiten.

**Comment by inverse** [comment]
@mafrei2019 I don't understand

**Comment by GameWatchProjects** [comment]
@inverse He means, that he activate Pi-Hole (runs by him in a docker container) as DNS, then Pi-Hole blocked Tasmoadmin v3.1.1 complete, when he deactivate Pi-Hole, Tasmoadmin is shown normal in the browser.

In my owen case i've ugraded to v.3.1.1 and the my Pi-Hole (runs on a Pi4 hardware in my network) show only very few entries (6-10 per day) in the query log, when i switch to v3.1.0 then the container runs "amok" (4-6 entry's per Minute) in the Pi-Hole query log.In my case the websites from Tasmoadmin is shown normal and and I never figured it out if I hadn't happened to look at the query log.

**Comment by mafrei2019** [comment]
Nein, nicht ganz,TasmoAdmin läuft ohne Probleme. PiHole sperrt die von dir genannten Seiten. Ich denke aber das liegt nicht an TA.

**Comment by inverse** [comment]
Are either if you still experiencing issues or shall I close this? I'm not sure what could be causing the issue and have been unable to reproduce.
